traffic_ops/testing/api/v3/traffic_control_test.go

```diff
@@ -31,6 +31,7 @@ type TrafficControl struct {
 	DeliveryServiceRequests                           []tc.DeliveryServiceRequest             `json:"deliveryServiceRequests"`
 	DeliveryServiceRequestComments                    []tc.DeliveryServiceRequestComment      `json:"deliveryServiceRequestComments"`
 	DeliveryServices                                  []tc.DeliveryServiceNullableV30         `json:"deliveryservices"`
+	DeliveryServicesV31                               []tc.DeliveryServiceV31                 `json:"deliveryservicesV31"`
```

Comment:
We probably wouldn't need this and the new additions to tc-fixtures.json either (assuming we alias like I mention in the first comment).